dotnet/test/AutoGen.OpenAI.Tests/MathClassTest.cs

```diff
@@ -46,38 +48,43 @@ public async Task<string> UpdateProgress(int correctAnswerCount)
         {
             if (correctAnswerCount >= 5)
             {
-                return $@"// ignore this line [UPDATE_PROGRESS]
+                return $@"[UPDATE_PROGRESS]
 {GroupChatExtension.TERMINATE}";
             }
             else
             {
-                return $@"// ignore this line [UPDATE_PROGRESS]
+                return $@"[UPDATE_PROGRESS]
 the number of resolved question is {correctAnswerCount}
 teacher, please create the next math question";
             }
         }
 
 
         [ApiKeyFact("AZURE_OPENAI_API_KEY", "AZURE_OPENAI_ENDPOINT")]
-        public async Task AssistantAgentMathChatTestAsync()
+        public async Task OpenAIAgentMathChatTestAsync()
         {
-            var teacher = await CreateTeacherAssistantAgentAsync();
-            var student = await CreateStudentAssistantAgentAsync();
             var key = Environment.GetEnvironmentVariable("AZURE_OPENAI_API_KEY") ?? throw new ArgumentException("AZURE_OPENAI_API_KEY is not set");
             var endPoint = Environment.GetEnvironmentVariable("AZURE_OPENAI_ENDPOINT") ?? throw new ArgumentException("AZURE_OPENAI_ENDPOINT is not set");
-            var model = "gpt-35-turbo-16k";
-            var admin = new GPTAgent(
-                name: "Admin",
-                systemMessage: $@"You are admin. You ask teacher to create 5 math questions. You update progress after each question is answered.",
-                config: new AzureOpenAIConfig(endPoint, model, key),
-                functions: new[]
-                {
-                    this.UpdateProgressFunction,
-                },
+
+            var openaiKey = Environment.GetEnvironmentVariable("OPENAI_API_KEY");
```

Comment:
Nit : remove `key` and `endpoint` ? Seems like openaikey is being used instead. Maybe the attribute tag `ApiKeyFact` might be changed as well?